src/components/hero/_hero.scss

```diff
@@ -36,6 +54,18 @@
                 width: 100%;
             }
         }
+        .ons-breadcrumbs__link {
+            color: var(--ons-color-text-inverse);
+
+            &:hover {
+                color: var(--ons-color-text-inverse);
+                text-decoration: underline solid var(--ons-color-text-inverse) 2px;
+            }
+            &:focus:hover {
+                color: var(--ons-color-text);
+                text-decoration: none;
+            }
+        }
```

Comment:
This is duplicated under the dark class, I think we should restructure this to remove the duplication